services/app-web/src/pages/StateSubmission/RateDetails/SingleRateFormFields.tsx

```diff
@@ -219,10 +219,22 @@ export const SingleRateFormFields = ({
                 <Label htmlFor={`${fieldNamePrefix}.rateProgramIDs`}>
                     What rates are included in this certification?
                 </Label>
-                <span className={styles.requiredOptionalText}>Required</span>
-                <span className={styles.requiredOptionalText}>
-                    This information will be used to generate the rate name
-                </span>
+                <div role="note">
+                    <span className={styles.requiredOptionalText}>
+                        Required
+                    </span>
+                    <span className={styles.requiredOptionalText}>
+                        This information will be used to generate the rate name
+                    </span>
+                    <LinkWithLogging
+                        aria-label="Managed care programs guidance (opens in new window)"
+                        href={'/help#managed-care-programs-guidance'}
+                        variant="external"
+                        target="_blank"
+                    >
+                        Contact the Help Desk to edit state programs list
```

Comment:
In the [designs](https://www.figma.com/design/frKNnm6lkcpdjJ2eVJE8Bj/Managed-Care-Review?node-id=9627-31925&t=KHchJx1Rj3uGnlIq-0), the language I see is 'Contact the help desk to edit this list`

Reply:
Good catch! I switched the copy text for state programs dropdown and rate programs dropdown.